core/src/main/java/edu/wpi/grip/core/serialization/SourceConverter.java

```diff
@@ -54,16 +54,16 @@ public Object unmarshal(HierarchicalStreamReader reader, UnmarshallingContext co
       // are in the process of loading.
       final Source source = sourceFactory.create(sourceClass, properties);
 
-      // Instead of returning the source, post it to the event bus so both the core and GUI
-      // classes know it
-      // exists.
-      eventBus.post(new SourceAddedEvent(source));
-
       // Now that the source has been added it needs to be initialized
       // We do it safely here in case the source has changed in some way out
       // of our control. For example, if a webcam is no longer available.
       source.initializeSafely();
 
+      // Instead of returning the source, post it to the event bus so both the core and GUI
+      // classes know it
+      // exists.
```

Comment:
Formatting.

Reply:
Heh. I moved the `initializeSafely()` line and comment, but the diff shows the event post lines changed instead.